### ISSUE TITLE
"Friends" are showed on the map of the Owntracks app

### ISSUE THREAD

**Issue by cedricdelecole** [owntracks/ios#818]
Hello, I post it as an issue because I think this is really abnormal ... I explain.

I have installed Owntracks on the iPhone device of my child (to be integrated and used on the Home assistant - this is not working, but this is another story ...).
On the map of Owntracks, I the family members's position together with the local device' position.

I am wondering how this is possible ... so to result my child sees on his app :
- his position
- My position
- The position of his 2 sisters
- The position of my wife.

None of us were requested to allow this (we even do not have Owntracks on our Android and IOS devices)... can you please clarify if this is an issue and how can this be possible ?

Many thanks .

**Comment by jpmens** [comment]
> None of us were requested to allow this (we even do not have Owntracks on our Android and IOS devices)

this is definitely something you should take up with Home Assistant or check in their documentation.

**Comment by jpmens** [comment]
There are exactly two possible mechanisms by which Friends data can appear in OwnTracks:
1. a friend's location publish is coming in via a configured MQTT connection (possibly from retained data which is on the MQTT broker)
2. a friend's location is coming in via a configured HTTP connection where the backend is **explicitly** providing a JSON array of friends which should be use on the client app

If you have not (previously) configured 1 or 2 but now have a HomeAssitant configuration then the data **must** be coming via them, possibly by some older configuration done some time ago?

**Comment by cedricdelecole** [comment]
No ... I installed it couple of dzys agi, and as I said even no entities are created under the owntracks intehration on HA as it should.

I am going to disable and uninstall that directly
;-)

**Comment by jpmens** [comment]
I think this has been [discussed here previously](https://github.com/owntracks/ios/issues/701#issuecomment-1021115174), but once again, we are probably not familiar enough with the HomeAssistant integration.

**Comment by jpmens** [comment]
I should probably also mention at this point that we've always been a bit "surprised" that HomeAssistant has not talked to us about their OwnTracks integration ... at least not as far as I recall.

**Comment by kfdm** [comment]
> I should probably also mention at this point that we've always been a bit "surprised" that HomeAssistant has not talked to us about their OwnTracks integration ... at least not as far as I recall.

Seems like there is no codeowner for the integration on their side

https://github.com/home-assistant/core/blob/5b2113c43da1d17dc09b47d23d5542d3860884cb/homeassistant/components/owntracks/manifest.json#L5

Though they do have labels to track that integration

https://github.com/home-assistant/core/labels/integration%3A%20owntracks
https://github.com/home-assistant/core/labels/integration%3A%20owntracks_http

(not suggesting this is something the Owntracks project is responsible to fix, was just curious myself so posting the links as reference)

**Comment by cedricdelecole** [comment]
Yeah ... I am a simple user of Home assistant, there are integration which are "public" I understand like for exemple the Owntracks ... and they are integrations done by individuals.

**Comment by jpmens** [comment]
@cedricdelecole have you been able to reset your apps and understand where the data is originating from?

**Comment by cedricdelecole** [comment]
I stopped using it and now using Icloud v3 instead
